modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/model/listener/APIEndpointRelevantObjectEntryModelListener.java

```diff
@@ -146,11 +149,24 @@ private void _validate(ObjectEntry objectEntry) {
 					_objectFieldLocalService.getObjectField(
 						objectEntry.getObjectDefinitionId(), "path");
 
+				String message = "";
+				String messageKey = "";
+
+				if (pathString.startsWith(StringPool.FORWARD_SLASH)) {
+					message =
+						"%s can have a maximum of 255 alphanumeric characters";
+					messageKey =
+						"x-can-have-a-maximum-of-255-alphanumeric-characters";
+				}
+				else {
+					message = "%s must start with the \"/\" character";
+					messageKey = "x-must-start-with-the-\"/\"-character";
+				}
+
 				throw new ObjectEntryValuesException.InvalidObjectField(
 					String.format(
-						"%s can have a maximum of 255 alphanumeric characters",
-						objectField.getLabel(user.getLocale())),
-					"x-can-have-a-maximum-of-255-alphanumeric-characters",
+						message, objectField.getLabel(user.getLocale())),
```

Comment:
Maybe we could create a variable with `objectField.getLabel(user.getLocale())` to avoid translating it more than once